packages/manager/apps/dedicated/client/app/ip/components/list/firewall/game/ip-ip-firewall-game.controller.js

```diff
@@ -294,6 +294,47 @@ export default /* @ngInject */ function IpGameFirewallCtrl(
     });
   }
 
+  function checkPortsNotUsed() {
```

Comment:
```suggestion
  function hasPortsAlreadyUsed() {
```
With the current name, we don't understand if having `true` as a result is OK or not without looking into the code function.
You might need to change the return value regarding your name for sure